dep.cc

```diff
@@ -534,7 +536,7 @@ class DepBuilder {
 
   shared_ptr<Rule> first_rule_;
   unordered_map<Symbol, DepNode*> done_;
-  unordered_set<Symbol> phony_;
+  set<Symbol> phony_;
```

Comment:
It does for gen_all_phony_targets:

    if (g_flags.gen_all_phony_targets) {
      for (Symbol s : phony_)
        targets.push_back(s);
    }